### DISCUSSION TITLE
Oculars - Adding Scope and Sensor is a NIGHTMARE!!!

### DISCUSSION THREAD

**Discussion by Dynan3** [Stellarium/stellarium#1630] [Ideas]
First, I DO want to thank you for all the time and energy you have put in to create a premier Astronomy Program! Outstanding! I use no other..no need...

But I am trying to add a Canon T5 with two lenses, an ASI533 camera, and a ES127 telescope to the Oculars for framing.

The process is entirely **_USER HOSTILE_**!

Suggest:

~ Be able to delete multiple unused sensors and telescopes in a group...not laboriously pecking each one.

~ Adding scope/sensor information only to see it disappear is disheartening. Seeing a new, default 'My CCD' appear instead of all the info I added is frustrating!

~ Should be able to EDIT an entry and 'ADD AS NEW'.

I've been using computers since 1986 and this is the most frustrating process in memory! Unless I'm missing something, which may well be the case, can you simplify the process?

Thanks again for all you do for the Astronomy Community!

**Comment by Dynan3** [comment]
I might mention that I'm trying to add the same equipment (DSLR/ASI183) with different rotation angles. Can't seem to add one with 0° and 90°.

**Reply by gzotti** [reply]
Admittedly, it takes a few seconds to get accustomed to the interface. However it has been like this for a good decade.
You press "new" and edit the values, give a name, etc.
Deleting the 20 default instruments may take another 20 seconds, yes, indeed. Once.

**Comment by Dynan3** [comment]
Apologies...I see that rotation angle cannot be set in Sensor data. (Why have it there?)
(You didn't need to be snarky.)

**Comment by Dynan3** [comment]
Actually I appreciate a good snark every once in a while...keeps the ego in check. Thanks.

(There is no "New" button, BTW. I assume you are saying to press any "MyCCD" listed.)

All is happy in AstroLand.

Again, Thanks to ALL for an outstanding astro program, and your time and dedication to it...

**Reply by gzotti** [reply]
Alright, not "New" but "Add". (I rarely use this plugin.) And you have to select the new entry before editing. Still not rocket science.

**Reply by Dynan3** [reply]
Still snarky...I'm learning a new plugin too. Good Day Sir.

**Comment by axd1967** [comment]
The ergonomy of the plugin is indeed not the best and it requires some experience and love to get used to it.

But you get used to it by working a lot with it and figure out how all buttons works. For example, add more objects, such as you smartphone FOV, binoculars, anything that you have.
